pkg/bindings/network/network.go

```diff
@@ -0,0 +1,51 @@
+package network
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/containernetworking/cni/libcni"
+	"github.com/containers/libpod/pkg/bindings"
+)
+
+func Create() {}
+func Inspect(ctx context.Context, nameOrID string) (map[string]interface{}, error) {
```

Comment:
Why use interface?  Everything has been string-ified to get here.  So you would need to convert them to xxx for this type of return.

Reply:
ah yes, the joys of working with cni ... they are interfaces ...